src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/CreditManager.cs

```diff
@@ -28,30 +28,32 @@ public CreditManager(IHttpTrace owner, string name, int initialCredit)
             _current = initialCredit;
         }
 
+        public int Current => Volatile.Read(ref _current);
```

Comment:
It looks like this is only used to see if any credit is available -- i.e. we don't care about the actual value, we just care that it's >0. Can we change this to something like `bool IsCreditAvailable()`?

Reply:
No, we actually read the value in [Http2Connection.AcquireStreamSlot()](https://github.com/dotnet/runtime/blob/18e14dc557f6022b56422fb2b1d04b38bb859d0e/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs#L153).